doc/api/tty.md

```diff
@@ -88,6 +88,17 @@ process.stdout.on('resize', () => {
 });
 ```
 
+*Note:* On Windows resize events will be emitted only if stdin is unpaused 
+(by a call to `resume()` or by adding a data listener) and in raw mode. It can 
+also be triggered if a terminal control sequence that moves the cursor is written 
+to the screen.
+
+*Note:* On Windows when changing console height the resize event will be 
```

Comment:
2. Sorry, but I'm not entirely sure what you want here. 

Can you paste the body of text you'd like? Or be very verbose about what should be replaced.